Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM apache/airflow:2.4.0-python3.10
+FROM apache/airflow:2.6.2-python3.11
 
-ENV POETRY_VERSION=1.2.0
+ENV POETRY_VERSION=1.5.0
```

Comment:
Is there a process to update the poetry version on some regular basis?

Reply:
That's a good question. I doubt we can automate (easily) since this is direct in the Dockerfile. I'm going to merge this but investigate how to approach this. :+1: